emails/shared/style.css

```diff
@@ -0,0 +1,79 @@
+body {
```

Comment:
Yep, global styles. If a template has styles specific to it, it can add its own style sheet its template folder and link to the stylesheet with `link(rel="stylesheet", href="/{template folder}/style.css", data-inline)` inside the `{template folder}/html.pug`